### DISCUSSION TITLE
PrimeVue Version Update Problems

### DISCUSSION THREAD

**Discussion by gunerfurkanilker** [primefaces/community#1403] [PrimeVue Templates]
Hello,

When i try to update my primevue project in order to use new components features and etc. my project is break on CSS. I bought PrimeVue Freya months ago and i cannot upgrade primevue version because upgrade messes up my code - especially CSS - entirely.
Is there any errorless update that i can do ?

My current version is 3.30.2 and i'm trying to update to version 3.49.1

Thanks.

**Comment by tugcekucukoglu** [comment]
Hello,

The new version of the templates is released this Monday. Could you try to use the latest version? Which template are you using?

**Reply by gunerfurkanilker** [reply]
Hello,

I'm using Freya template, PrimeVue version of 3.30.2. In order to use newest version of PrimeVue and its components what do i have to do ? Is there any update or migration guide for existing Freya project ? Reason is i want to use some of components in my existing Freya project.

Thanks.
